soccer/gameplay/plays/offense/distraction1.py

```diff
@@ -0,0 +1,188 @@
+import robocup
+import standard_play
+import behavior
+import constants
+import main
+import skills.move
+import skills.capture
+import enum
+import evaluation
+import tactics.coordinated_pass
+import tactics.defense
+import play
```

Comment:
This is really picky, but I like to organize the imports like [adaptive formations](https://github.com/RoboJackets/robocup-software/blob/master/soccer/gameplay/plays/offense/adaptive_formation.py#L1). It makes it super easy to figure out what's been imported or not.